concourse/pipelines/templates/dev_build_pipeline-tpl.yml

```
@@ -12,6 +12,9 @@
 {# define whether multinode testing is requested for given EL OS versions #}
 {% set requested_multi = {'7': multinode_el7, '8': multinode_el8, '9': multinode_el9} %}
 
+{# define whether external pxf testing is requested for given Greenplum versions #}
+{% set requested_external_pxf = {'6': gp6_cli, '7': gp7_cli} %}
```

```suggestion
{% set requested_cli = {'6': cli_gp6, '7': cli_gp7} %}
```
here we are requesting CLI tests for given GP versions as semantically specified by variable names (GP6_CLI and GP7_CLI). The fact that they will be run with external PXF or not is decided internally.

Also, maybe we should switch the order in the names to list the feature first and then version it applies to last